### PR TITLE
Add company model on Flutter side

### PR DESCRIPTION
**Pull request by mozhaiskyi**

### CODE REVIEW

File: intercom_flutter/lib/company.dart [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    this.id,
+    this.name,
+    this.customAttributes,
+    this.createdAt,
+    this.monthlySpend,
+    this.plan,
+  });
+}
+
+Map<String, dynamic> mapToNativeParameters(Company company) {
```
Comment (lifedemons) [inline intercom_flutter/lib/company.dart:24]: could we use `=>` instead of `return`?

File: intercom_flutter/test/intercom_flutter_test.dart [hunk 2]
```diff
@@ -157,13 +157,17 @@ void main() {
     });
 
     test('updateUser', () {
+      final company = Company(
+        id: 'some_company_llc',
+        name: 'Soma Company LLC',
```
Comment (lifedemons) [inline intercom_flutter/test/intercom_flutter_test.dart:162]: typo, `Some`

Comment (mozhaiskyi) [reply intercom_flutter/test/intercom_flutter_test.dart:162]: danke!

File: intercom_flutter/lib/company.dart (hunk 1 above)
Comment (mozhaiskyi) [reply intercom_flutter/lib/company.dart:24]: Not sure. I didn't find any usage of `=>` in the same cases. Let's keep the project code style.

**Review by lifedemons — APPROVED** [review]

**Pull request merged**